userguide/snippets/x_rails.rst

```diff
@@ -23,25 +22,26 @@ Install a spring on the silver posts in the side of each rail.
 .. figure:: images/tn_x_spring.png
    :width: 60%
 
-
 Open the clamp latches on the ends of each rail. Place the rail in the
 rack with the front end toward the front of the rack, aligning the
 pins on both ends of the rail with the mounting holes in the rack.
 Swing the clamp latch closed to hold the rail in place. Use two of the
 supplied screws to secure the back end of the rail in place. Repeat
 the process for the second rail.
 
-
 .. figure:: images/tn_x_railclamp.png
    :width: 100%
 
+.. caution:: The unit weighs about 30 pounds.
```

Comment:
Same as above, exact weight from the spec sheet says 26 lbs unloaded, 44 lbs loaded.